shared/actions/plan-billing.js

```diff
@@ -173,17 +172,6 @@ function * fetchBillingAndQuotaSaga (): SagaGenerator<any, any> {
 
 function * bootstrapDataSaga (): SagaGenerator<any, any> {
   const billingStateSelector = ({planBilling}: TypedState) => planBilling
-  const overallBootstrappedSelector = ({config: {bootStatus}}: TypedState) => bootStatus
-  const loggedInSelector = ({config: {loggedIn}}: TypedState) => loggedIn
-
-  let bootstrapStatus: BootStatus = ((yield select(overallBootstrappedSelector)): any)
-  let loggedIn: boolean = ((yield select(loggedInSelector)): any)
-
-  while (bootstrapStatus !== 'bootStatusBootstrapped' || !loggedIn) {
-    yield call(delay, 500)
-    bootstrapStatus = ((yield select(overallBootstrappedSelector)): any)
-    loggedIn = ((yield select(loggedInSelector)): any)
-  }
 
```

Comment:
Shouldn't you check that you are logged in & bootstrapped, and if you aren't you wait for the logged in and bootstrap action before making the call ?